### PR TITLE
Add Home hero module and typography

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- **NOTE: Actual fonts are not in!**
- Adds Home Hero Module
- Transitions will come later because Maria and I need to discuss how it will work. 
- Typography classes
- Adds more boilerplate like `Button`, `Img`, and some schema fields like `buttonField`, `portableText, etc. 
- Sets up some of the major schemas: `Company` and `Founder`. 

### Type(s) of changes

- [x] New feature

### Motivation for PR

#9 

### Applicable screenshots:

image

### Follow-up PR

#11 to add overlay functionality (when you hover over a founder, their image will appear).

### CODE REVIEW

File: src/styles/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+/** Styles */
+
+.font-italic {
+  font-style: italic;
+}
+
+/** Sizes */
+
+.primary-xxl {
+  font-family: $benton;
```
Comment (chrismekim) [inline src/styles/_typography.scss:27]: Decided to use the names from the design files! Primary font, secondary font, and secondary bold font.

File: cms/schemas/layouts/homeHeroModule.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      validation: (Rule: any) => Rule.required(),
+    },
+    {
+      name: 'secondLine',
+      title: 'Second Line of Headline Text',
+      description: 'This text is the second line of the headline copy.',
+      type: 'string',
+      fieldset: 'copy',
+      validation: (Rule: any) => Rule.required(),
+    },
```
Comment (chrismekim) [inline cms/schemas/layouts/homeHeroModule.ts:32]: FYI this "firstLine" and "secondLine" will probably change in the future depending on the transition changes I talk about with Maria, but keeping it here for now!

File: cms/schemas/fields/buttonField.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  title: 'Button',
+  name: 'buttonField',
+  type: 'object',
+  fields: [
+    {
+      title: 'Label',
+      name: 'label',
+      type: 'string',
+    },
+    {
```
Comment (mokaymokay) [inline cms/schemas/fields/buttonField.ts:11]: add aria label field maybe?

**Review by mokaymokay — APPROVED** [review]
LGTM, i don't think the schema files need to be typescript though

**Review by limsohee1002 — APPROVED** [review]

**Pull request merged**